@here/harp-mapview/lib/MapView.ts

```diff
@@ -1471,11 +1476,29 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Get distance from camera to the point of focus in world units.
+     *
+     * @note If camera does not point to any ground anymore the last focus point distance is
+     * then returned.
+     *
+     * @returns Last known focus point distance.
      */
     get lookAtDistance(): number {
         return this.m_lookAtDistance;
     }
 
+    /**
+     * Get world coordinates of camera focus point.
+     *
+     * @note The focus point coordinates are updated with each camera update so you don't need
+     * to re-calculate it, although if the camera started looking to the void, the last focus
+     * point is stored.
+     *
+     * @returns world coordinates of last camera focus point.
+     */
+    get lookAtPoint(): THREE.Vector3 {
```

Comment:
This should be called target. Se also: https://github.com/heremaps/harp.gl/pull/1027

Reply:
Ok.